db/db_tunables.h

```diff
@@ -1977,6 +1977,10 @@ REGISTER_TUNABLE("client_abort_on_slow",
                  "Enable watchdog to abort if a \"slow\" client is detected."
                  "  (Default: off)", TUNABLE_BOOLEAN, &gbl_client_abort_on_slow,
                  EXPERIMENTAL | INTERNAL, NULL, NULL, NULL, NULL);
+REGISTER_TUNABLE("fake_sc_replication_timeout",
```

Comment:
[`debug_switches.c`](https://github.com/bloomberg/comdb2/blob/master/util/debug_switches.c) seems a better place for the tunable since it's for debugging purpose only.